appcli/orchestrators.py

```diff
@@ -96,7 +96,29 @@ def task(
             cli_context (CliContext): The current CLI context.
             container_options (ContainerRuntimeOptions) : List of options to apply to the container.
             service_name (str): Name of the container to run.
-            extra_args (Iterable[str]): Extra arguments for running the container
+            extra_args (Iterable[str]): Extra arguments for running the container.
+
+        Returns:
+            CompletedProcess: Result of the orchestrator command.
+        """
+        raise NotImplementedError
+
+    def exec(
+        self,
+        cli_context: CliContext,
+        service_name: str,
+        command_name: str,
```

Comment:
Does it take a single command, with extra args? Or does this take a list of string representing the full command to exec?
Do we need both arguments?

Reply:
Signature has been changed to accept both command and args as a single List[str]